packages/block-editor/src/components/media-flow/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	FormFileUpload,
+	MenuItem,
+	Toolbar,
+	withNotices,
+} from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BlockControls from '../block-controls';
+import MediaUpload from '../media-upload';
+import MediaUploadCheck from '../media-upload/check';
+import LinkEditor from '../url-popover/link-editor';
+import LinkViewer from '../url-popover/link-viewer';
+
+const MediaFlow = (
+	{
+		mediaUpload,
+		mediaURL,
+		allowedTypes,
+		onSelect,
+		onSelectURL,
+		noticeOperations,
+		name = __( 'Replace' ),
+		multiple = false,
+	}
+) => {
+	const [ showURLInput, setshowURLInput ] = useState( false );
+	const [ showEditURLInput, setshowEditURLInput ] = useState( false );
```

Comment:
Typo here an on the line above, `setshowEditURLInput` should be `setShowEditURLInput`